scapy/layers/sctp.py

```diff
@@ -280,15 +312,113 @@ class SCTPChunkParamSupportedAddrTypes(_SCTPChunkParam, Packet):
                              4, padwith=b"\x00"), ]
 
 class SCTPChunkParamECNCapable(_SCTPChunkParam, Packet):
-    fields_desc = [ ShortEnumField("type", 32768, sctpchunkparamtypes),
+    fields_desc = [ ShortEnumField("type", 0x8000, sctpchunkparamtypes),
                     ShortField("len", 4), ]
 
+class SCTPChunkParamRandom(_SCTPChunkParam, Packet):
+    fields_desc = [ ShortEnumField("type", 0x8002, sctpchunkparamtypes),
+                    FieldLenField("len", None, length_of="random",
+                                  adjust = lambda pkt,x:x+4),
+                    PadField(StrLenField("random", os.urandom(32),
```

Comment:
You probably want to replace `os.urandom(32)` with `RandBin(32)`, for at least two reasons:
- `RandBin()` and other volatile values exist in Scapy for this purpose.
- With your code, the random string would be generated when the module is loaded and will be the same for every packet within the same Scapy session or program execution.

You would have to import `RandBin` from `scapy.volatile`, and remove the `import os`.

What do you think?